app/src/main/res/layout/activity_get_details.xml

```diff
@@ -49,197 +65,260 @@
                     android:layout_marginTop="20dp"
                     android:layout_marginRight="60dp"
                     android:layout_weight="3"
-                    android:background="@drawable/button_background"
+                    android:background="@drawable/btn_bg"
                     android:minHeight="40dp"
                     android:text="Get Details"
+                    android:textSize="16sp"
+                    android:textColor="@color/white"
+                    android:fontFamily="@font/avenir_bold"
+                    android:textAllCaps="false"
                     tools:layout_editor_absoluteX="117dp"
                     tools:layout_editor_absoluteY="107dp" />
 
             </LinearLayout>
 
-            <LinearLayout
-                android:id="@+id/details_layout"
```

Comment:
Here! You removed the details_layout Id which is related connected to java file. Please assign this id to the relevant layout.

Reply:
Alright. change it right away

Reply:
updated